index.src.html

```diff
@@ -4133,6 +4136,42 @@
 : <dfn event for=ImageTrack>change</dfn>
 :: Fired at the {{ImageTrack}} when the {{ImageTrack/frameCount}} is altered.
 
+Resource Reclamation{#resource-reclamation}
+==============================================
+
+When resources are constrained, a User Agent might want to pro-actively reclaim
+codecs. This is particularly true in the case where hardware codecs are limited,
+and shared accross all web pages.
+
+A User Agent is allowed to [=reclaim a codec=], freeing the resources of a
+<dfn>reclaimable codec</dfn>, which is a defined as a codec that is either an
+[=inactive codec=] or a [=background codec=].
```

Comment:
After this sentence, should we explicitly ban the reclaiming of active & foreground codecs. e.g. 

`A user agent must not reclaim a codec that is not meeting one or both of these criteria.`

Reply:
Maybe "A User Agent may only [=reclaim a codec=] that is either an [=inactive codec=] or a [=background codec=].  An [=active codec=] or [=foreground codec=] MUST NOT be reclaimed."

Reply:
I've made another pass. I am dropping the [=reclaimable codec=] term, as not all that useful.

I tried to avoid defining [=active codec=] and [=foreground codec=], since they are only used once. I'm happy to change this to add explicit definitions.